src/main/java/dk/cngroup/lentils/service/StatusService.java

```diff
@@ -30,20 +33,24 @@ public void markCypher(final Cypher cypher, final Team team, final CypherStatus
     }
 
     public int getStatusScore(final Team team, final Cypher cypher) {
-        return getStatusByTeamAndCypher(team, cypher).getCypherStatus().getStatusValue();
+        if (getStatusByTeamAndCypher(team, cypher) == null) {
+            return 0;
+        } else {
+            return getStatusByTeamAndCypher(team, cypher).getCypherStatus().getStatusValue();
+        }
     }
 
     private Status getStatusByTeamAndCypher(final Team team, final Cypher cypher) {
         Status status = statusRepository.findByTeamAndCypher(team, cypher);
-        if (status == null) {
```

Comment:
Tohle tady nechej a misto ifu pouzij try/catch.

Reply:
pokud pouziju try / catch tak to spadne - nullpointer exception

Reply:
hotovo